lib/Language/Souffle/Experimental.hs

```diff
@@ -92,6 +93,7 @@ module Language.Souffle.Experimental
   , __
   , underscore
   , (|-)
+  , (||-)
```

Comment:
Naming is hard, let's go shopping!

I think if this is finished, you can just use `|-` for the new one, and get rid of the old one everywhere.